pandas/io/tests/test_packers.py

```diff
@@ -496,6 +517,46 @@ def test_compression_blosc(self):
             assert_frame_equal(self.frame[k], i_rec[k])
 
 
+class TestMsgpack():
```

Comment:
the reason for inheritance is for us to potentially control testing options (eg to have deprecation warnings raise errors on certain runs and things like that ) - the pickle one is the only class that doesn't inherit and I suppose for the yield syntax reason

though really like it too